src/app.ts

```diff
@@ -166,7 +166,40 @@ export class App extends Construct {
       default:
         break;
     }
+
+  }
+
+  // return string with all yaml objects inside app
+  public synthYaml(): any {
```

Comment:
Use proper docstrings:

```suggestion
  /**
   * Synthesizes the app into a YAML string.
   *
   * @returns A string with all YAML objects across all charts in this app.
   */
  public synthYaml(): any {
```